bridge_ui/src/utils/consts.ts

```diff
@@ -275,6 +275,20 @@ export const SOLANA_HOST = process.env.REACT_APP_SOLANA_API_URL
   ? clusterApiUrl("devnet")
   : "http://localhost:8899";
 
+export const ETH_RPC_HOST =
+  CLUSTER === 'mainnet'
+    ? ''
+    : CLUSTER === 'testnet'
+    ? 'https://ethereum-goerli.publicnode.com'
+    : 'http://localhost:8545'
+
+export const BSC_RPC_HOST =
+  CLUSTER === 'mainnet'
+    ? ''
+    : CLUSTER === 'testnet'
+    ? 'https://data-seed-prebsc-1-s1.binance.org:8545'
+    : 'http://localhost:8546'
```

Comment:
We need to query both the source chain and target chain simultaneously to determine the status of a transaction. We can check if the transaction is `Pending` or `Confirmed` based on the height of the source chain. If the transaction has been confirmed, we need to query the target chain's contract to check if the transaction has been completed.

We want these query requests to be handled through the wallet. However, it doesn't work if both the source chain and target chain are EVM chains because MetaMask can only connect to one EVM chain at a time. That's why I added `ETH_RPC_HOST` and `BSC_RPC_HOST`. In this case, the requests for the source chain will be handled through the wallet, while the requests for the target chain will be handled through the RPC host.

For the mainnet, we will run our own full node, so that shouldn't be a problem. However, for the testnet, I'm not sure if these two RPC hosts will be sufficient.